BUILDING.md

```diff
@@ -46,6 +46,24 @@ make setup -j$(nproc) OPTFLAGS=-O2 DEBUG=0
 make -j $(nproc) OPTFLAGS=-O2 DEBUG=0
 ```
 
+## macOS
+
+1. Requires `gcc@12, sdl2, libpng, glew, x11` (can be installed via brew, etc)
+```bash
+# Clone the repo
+git clone https://github.com/HarbourMasters/Shipwright.git
+cd ShipWright
+# Copy the baserom to the OTRExporter folder
+cp <path to your ROM> OTRExporter
+
+cd soh
+# Extract the assets/Compile the exporter/Run the exporter
+# -jX defines number of cores to use for compilation - lower or remove entirely if having issues
+make setup -j8 DEBUG=0 CC=gcc-12 CXX=g++-12
```

Comment:
-jx no longer works for me for make setup. Does it work for you?

Reply:
It does! But no code that we change would affect that, that is a parameter to the `make` command

Reply:
Thanks, it just worked now :-S